### PR TITLE
Fix typos all around

### PR DESCRIPTION
**Pull request by szepeviktor**
Found few misspellings.

### CODE REVIEW

**Comment by szepeviktor** [comment]
Should I revert "Invokable"?

**Comment by jcastroa87** [comment]
Wow, thanks a lot @szepeviktor so many small typos 🙈

I will ask @pxpm to check and merge if everything is ok.

Thanks again.

**Comment by szepeviktor** [comment]
Glad to contribute.

You could introduce https://github.com/crate-ci/typos to any of your software projects.

File: src/app/Library/CrudPanel/Traits/Fields.php [hunk 1]
```diff
@@ -505,12 +505,12 @@ public function getStrippedSaveRequest($request)
             return $setting($request);
         }
 
-        // if an invokable class was passed
+        // if an invocable class was passed
```
Comment (pxpm) [inline src/app/Library/CrudPanel/Traits/Fields.php:508]: Let's revert this please. In php it's invokable classes. 🙏

File: src/app/Library/CrudPanel/Traits/Fields.php [hunk 2]
```diff
@@ -505,12 +505,12 @@ public function getStrippedSaveRequest($request)
...
         // eg. \App\Http\Requests\BackpackStrippedRequest
         if (is_string($setting) && class_exists($setting)) {
             $setting = new $setting();
 
-            return is_callable($setting) ? $setting($request) : abort(500, get_class($setting).' is not invokable.');
+            return is_callable($setting) ? $setting($request) : abort(500, get_class($setting).' is not invocable.');
```
Comment (pxpm) [inline src/app/Library/CrudPanel/Traits/Fields.php:513]: Let's revert this please. In php it's invokable classes. 🙏

File: src/config/backpack/operations/create.php [hunk 3]
```diff
@@ -37,6 +37,6 @@
 
     // Before saving the entry, how would you like the request to be stripped?
     //  - false - use Backpack's default (ONLY save inputs that have fields)
-    //  - invokable class - custom stripping (the return should be an array with input names)
+    //  - invocable class - custom stripping (the return should be an array with input names)
```
Comment (pxpm) [inline src/config/backpack/operations/create.php:40]: Let's revert this please. In php it's invokable classes. 🙏

File: src/config/backpack/operations/form.php [hunk 4]
```diff
@@ -34,6 +34,6 @@
 
     // Before saving the entry, how would you like the request to be stripped?
     //  - false - use Backpack's default (ONLY save inputs that have fields)
-    //  - invokable class - custom stripping (the return should be an array with input names)
+    //  - invocable class - custom stripping (the return should be an array with input names)
```
Comment (pxpm) [inline src/config/backpack/operations/form.php:37]: Let's revert this please. In php it's invokable classes. 🙏

File: src/config/backpack/operations/update.php [hunk 5]
```diff
@@ -37,6 +37,6 @@
 
     // Before saving the entry, how would you like the request to be stripped?
     //  - false - use Backpack's default (ONLY save inputs that have fields)
-    //  - invokable class - custom stripping (the return should be an array with input names)
+    //  - invocable class - custom stripping (the return should be an array with input names)
```
Comment (pxpm) [inline src/config/backpack/operations/update.php:40]: Let's revert this please. In php it's invokable classes. 🙏

File: tests/Unit/CrudPanel/CrudPanelFieldsTest.php [hunk 6]
```diff
@@ -626,15 +626,15 @@ public function testGetStrippedSaveRequestWithClass()
...
+            Invocable::class,
             'update'
         );
         $this->crudPanel->setOperation('update');
         $this->crudPanel->setModel(User::class);
         $request = request()->create('/users/1/edit', 'POST', ['name' => 'john']);
         $result = $this->crudPanel->getStrippedSaveRequest($request);
         $this->assertIsArray($result);
-        $this->assertSame(['invokable' => 'invokable'], $result);
+        $this->assertSame(['invocable' => 'invocable'], $result);
```
Comment (pxpm) [inline tests/Unit/CrudPanel/CrudPanelFieldsTest.php:637]: Let's revert this please. In php it's invokable classes. 🙏

File: tests/Unit/CrudPanel/CrudPanelFieldsTest.php [hunk 7]
```diff
@@ -900,10 +900,10 @@ public function testCheckReturnTypesForWhenInferingRelation()
     }
 }
 
-class Invokable
+class Invocable
```
Comment (pxpm) [inline tests/Unit/CrudPanel/CrudPanelFieldsTest.php:903]: Let's revert this please. In php it's invokable classes. 🙏

File: tests/Unit/CrudPanel/CrudPanelFieldsTest.php [hunk 8]
```diff
@@ -900,10 +900,10 @@ public function testCheckReturnTypesForWhenInferingRelation()
...
 {
     public function __invoke(): array
     {
-        return ['invokable' => 'invokable'];
+        return ['invocable' => 'invocable'];
```
Comment (pxpm) [inline tests/Unit/CrudPanel/CrudPanelFieldsTest.php:907]: Let's revert this please. In php it's invokable classes. 🙏

**Review by pxpm — CHANGES_REQUESTED** [review]
Thanks @szepeviktor , I just marked all the places where `invocable` should be reverted so that I can go back and check if all of them were properly reverted, I've just did a quick look at the docs of the tool you mentioned and most likely you can just add the word to the ignore list 🙏 

I will sure be having a better look at that tool, thanks for sharing and for the PR, much appreaciated 🙏

**Comment by szepeviktor** [comment]
Done 🍏

**Pull request merged**